src/DependencyInjection/Configuration.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * Node Bundle for Contao Open Source CMS.
+ *
+ * @copyright  Copyright (c) 2019, terminal42 gmbh
+ * @author     terminal42 <https://terminal42.ch>
+ * @license    MIT
+ */
+
+namespace Terminal42\NodeBundle\DependencyInjection;
+
+use Symfony\Component\Config\Definition\Builder\TreeBuilder;
+use Symfony\Component\Config\Definition\ConfigurationInterface;
+
+final class Configuration implements ConfigurationInterface
+{
+    public function getConfigTreeBuilder(): TreeBuilder
+    {
+        $treeBuilder = new TreeBuilder('terminal42_node');
+        $treeBuilder
+            ->getRootNode()
+            ->children()
+                ->booleanNode('geoip2country')
+                    ->defaultTrue()
```

Comment:
In `terminal42/contao-geoip2-country` there is a configuration for what DCA tables to support (https://github.com/terminal42/contao-geoip2-country#bundle-configuration). This basically mimics this configuration for nodes.

However, the configuration in geoip2country is mainly meant for an app developer to register additional tables, maybe it's not necessary here because there are user permissions? 🤷

Reply:
Yeah well, I mean. It doesn't harm either so feel free to merge :)